### ISSUE TITLE
Signed integer is greater than maximum

### ISSUE THREAD

**Issue by JoshVStaden** [harvardnlp/botnet-detection#3]
Hi

I seem to be having an issue downloading from the 'chord' dataset. When I run the line

`botnet_dataset_train = BotnetDataset(name='chord', split='train', graph_format='dict')`

I seem to get this message:

Downloading` https://zenodo.org/record/3689089/files/botnet_chord.tar.gz
Traceback (most recent call last):
  File "train_lstm.py", line 4, in <module>
    botnet_dataset_train = BotnetDataset(name='chord', split='train', graph_format='dict')
  File "/home/joshuavanstaden/.virtualenvs/cvML/lib/python3.8/site-packages/botdet-0.1.0-py3.8.egg/botdet/data/dataset_botnet.py", line 64, in __init__
    self.download()
  File "/home/joshuavanstaden/.virtualenvs/cvML/lib/python3.8/site-packages/botdet-0.1.0-py3.8.egg/botdet/data/dataset_botnet.py", line 133, in download
    path = download_url(self.url, self.raw_dir)
  File "/home/joshuavanstaden/.virtualenvs/cvML/lib/python3.8/site-packages/botdet-0.1.0-py3.8.egg/botdet/data/url_utils.py", line 54, in download_url
    f.write(data.read())
  File "/usr/lib/python3.8/http/client.py", line 471, in read
    s = self._safe_read(self.length)
  File "/usr/lib/python3.8/http/client.py", line 612, in _safe_read
    data = self.fp.read(amt)
  File "/usr/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.8/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.8/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
OverflowError: signed integer is greater than maximum

Let me know if you need any more info. TIA.

**Comment by jzhou316** [comment]
Hi @JoshVStaden I'll take a look and get back to you later today

**Comment by jzhou316** [comment]
Hi @JoshVStaden sorry for the delay. I tested the command on my side and it works fine (python 3.7 in my case). Could it be a package/environment issue? Or the memory size being small? I searched for similar issues and maybe the following might give more information 
https://bugs.python.org/issue42853
https://stackoverflow.com/questions/65667843/zipfile-how-to-download-a-zipfile-and-save-without-extract
